.lagoon.yml

```diff
@@ -4,9 +4,21 @@ project: dpl-cms-core
 
 ssh: 20.238.147.183:22
 api: https://api.lagoon.dplplat01.dpl.reload.dk/graphql
+environment_variables:
+  git_sha: 'true'
 
 tasks:
   post-rollout:
+    - run:
+        name: Create new GH deployment
+        command: |
+          # @TODO Where can i place the token as a secret?
```

Comment:
Consider mentioning that we do not use `set -e` as we do not want the build process to stop if deployment handling fails.